message/model.json

```diff
@@ -1,8 +1,11 @@
 {
+
```

Comment:
other json files don't have this blank line - any reason you added it?

Reply:
No, it just came through rearranging, I'll remove it tomorrow.